client/src/components/profile/memberscard.js

```diff
@@ -0,0 +1,127 @@
+import withStyles from '../jss'
+import { getMembers, addMember, removeMember } from '../../api/members'
+import { updateAccount } from '../../api/profile'
+import { useState, useCallback, useEffect } from 'preact/hooks'
+import config from '../../../../config/client'
+
+import Form from '../form'
+import { withToast } from '../toast'
+
+const MemberRow = ({ id, name, setMembers }) => {
+  const handleDelete = useCallback(() => {
+    removeMember({ id })
+      .then(() => {
+        setMembers(members => members.filter(a => a.id !== id))
+      })
+  }, [id, setMembers])
+
+  return (
+    <div class='u-vertical-center' style='width: 100%; display: flex; justify-content: space-between;' key={id}>
```

Comment:
style this with JSS? (and drop the `u-vertical-center` as well since we might as well set `align-items` ourselves, given that we’re explicitly specifying half the styles applied by that utility class already)